platform/app/cypress/integration/volume/MPR.spec.js

```diff
@@ -131,4 +119,19 @@ describe('OHIF MPR', () => {
         );
       });
   });
+
+  it('should activate window level when the active Crosshairs tool for MPR is clicked', () => {
```

Comment:
shouldn't it carry over the previously active tool to MPR (if it exists)?

Reply:
Not how it was implemented... there is simply a default tool to fall back to.

Reply:
But I fixed that I'm pretty sure that when you have a tool in deafult HP it gets transferred to MPR too

Reply:
This test is testing the new functionality whereby if the crosshairs tool is active and then clicked, it will deactivate the crosshairs tool and make window level the the active tool. It is NOT testing which tool is carried over to MPR.